### PR TITLE
Feat(Debug Misc): Added Options

### PR DESCRIPTION
**Pull request by TheGreenBandit**
Saw the tab empty and thought why not fill it. If anyone has anything to add comment.

### CODE REVIEW

**Comment by Yimura** [comment]
How are these features related to locals? They fit better under the Misc tab.

**Comment by TheGreenBandit** [comment]
Skip cutscene maybe, i just put in the ones that are clientside

**Comment by Yimura** [comment]
The locals tab is meant to debug script locals and does to not relate to anything client side in the way you describe it.

**Comment by TheGreenBandit** [comment]
> The locals tab is meant to debug script locals and does to not relate to anything client side in the way you describe it.

Ah alright, yeah i saw it totally different than it was my bad, ill just move the additions to misc and they can be used there

**Review by Yimura — APPROVED** [review]

File: src/views/debug/view_debug_locals.cpp [hunk 1]
```diff
@@ -1,12 +1,14 @@
 #include "view_debug.hpp"
+#include "gui/components/components.hpp"
+#include "natives.hpp"
```
Comment (Yimura) [inline src/views/debug/view_debug_locals.cpp]: Removed unused includes.

**Review by Yimura — CHANGES_REQUESTED** [review]

**Review by Yimura — APPROVED** [review]

**Pull request merged**